manga_auto_translator/segmentation/UNet.py

```diff
@@ -0,0 +1,156 @@
+# import the necessary packages
+
+
+from torch.nn import ConvTranspose2d
+from torch.nn import Conv2d
+from torch.nn import MaxPool2d
+from torch.nn import Module
+from torch.nn import ModuleList
+from torch.nn import ReLU
+from torchvision.transforms import CenterCrop
+from torch.nn import functional as F
+import torch
+import cv2
+import numpy as np
+
+class Block(Module):
+	def __init__(self, inChannels, outChannels):
+		super().__init__()
+		# store the convolution and RELU layers
```

Comment:
Can we remove most of the comments that are "identity functions" to the code right under them?